beatmap_reader/unit_tests/test_beatmap.py

```diff
@@ -144,6 +145,24 @@ def test_beatmap_loading_weird(self):
         self.assertEqual(distorted.tdata[0][Hitobject.TDATA_Y], 272)
 
 
+    def test_beatmap_loading_slider_ticks(self):
+        beatmap = BeatmapIO.open_beatmap(os.path.join(
+            'beatmap_reader', 'unit_tests', 'maps',
+            'osu', 'abraker - unknown (abraker) [slider_test].osu'
+        ))
+        data_path = os.path.join(
+            'beatmap_reader', 'unit_tests', 'data', 'slider_test_data.txt'
+        )
+        with open(data_path, 'r') as f:
+            data = ast.literal_eval(f.read())
```

Comment:
Executing what is intended to be a data file as a python script is *really* weird. This is supposed to be a json file.